README.md

```diff
@@ -130,11 +130,11 @@ docker run \
   -ti registry.cncf.ci/cncf/cross-cloud/provisioning:ci-stable-v0-2-0
 ```
 
-#### Quickstart for Packet.net
+#### Quickstart for metal.equinix.net
```

Comment:
Change request: metal.equinix.net should be metal.equinix.com

Reply:
https://github.com/crosscloudci/cross-cloud/pull/190/commits/349120db27a6480fee2eb1ca9c1c4e2f3317dcc7